docs/template_plugin/tests/functional/op_reference/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -0,0 +1,250 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace reference_tests;
+
+struct ExperimentalROIFunctional {
+    std::shared_ptr<Function> create_function(const std::vector<Tensor>& ed_inputs,
+                                              const std::vector<Tensor>& results,
+                                              const std::vector<Shape>& input_shapes,
```

Comment:
Why do we need to have shapes and element_type here?
Tensors already have shapes and element types.

Reply:
Fixed.